conjureup/controllers/destroy/gui.py

```diff
@@ -0,0 +1,38 @@
+from conjureup import controllers, juju
+from conjureup.app_config import app
+from conjureup.telemetry import track_exception, track_screen
+from conjureup.ui.views.destroy import DestroyView
+
+
+class Destroy:
+
+    def __init__(self):
+        self.view = None
+
+    def __handle_exception(self, exc):
+        track_exception(exc.args[0])
+        app.ui.show_exception_message(exc)
+
+    def finish(self, controller, model):
+        return controllers.use('destroyconfirm').render(controller, model)
+
+    def render(self):
+        models_map = {}
+        existing_controllers = juju.get_controllers()['controllers']
+        for cname, d in existing_controllers.items():
+            models_map[cname] = juju.get_models(cname)
```

Comment:
I think we should probably filter out the 'controller' model from the list of things you can kill.

If we want to enable killing off the whole controller, we should make that more obvious, and just use destroy-controller…